### PR TITLE
feat: add revocation to access-client and w3up-client

### PR DESCRIPTION
**Pull request by travis**
Add a new method on `access-client`'s `Agent` to make this work and call into it from `w3up-client`'s `Client`.

### CODE REVIEW

File: packages/access-client/test/agent.test.js
Comment (alanshaw) [inline packages/access-client/test/agent.test.js:271]: Is this seconds or milliseconds? The interface comment says seconds.

**Review by alanshaw — APPROVED** [review]
Looks ok to me but please resolve the question on the timestamp before merge 🙂

**Review by Gozala — CHANGES_REQUESTED** [review]
looks good but there's on edge case to handle

File: packages/access-client/test/agent.test.js
Comment (travis) [reply packages/access-client/test/agent.test.js:271]: oh I'm sure the interface comment is right! I saw `number`s in the types and didn't check, will fix

File: packages/access-client/src/agent.js
Comment (travis) [reply packages/access-client/src/agent.js:221]: happy to defer to your aesthetics here, done!

File: packages/access-client/src/agent.js
Comment (travis) [reply packages/access-client/src/agent.js]: ah good catch thanks! done

File: packages/access-client/test/agent.test.js
Comment (travis) [reply packages/access-client/test/agent.test.js:327]: done and done

File: packages/capabilities/src/types.ts
Comment (travis) [reply packages/capabilities/src/types.ts]: ah good call, done!

**Pull request merged**
